### PR TITLE
updateText

### PR DESCRIPTION
**Pull request by sohaibrachi**

### CODE REVIEW

File: src/components/Footer/Footer.css [hunk 1]
```diff
@@ -0,0 +1,26 @@
+.footer {
```
Comment (yazfield) [inline src/components/Footer/Footer.css:1]: why did you create css files?

Comment (sohaibrachi) [reply src/components/Footer/Footer.css:1]: i pressed on live compilation of  scss to css

**Review by sohaibrachi — COMMENTED** [review]
i pressed on live compilation of scss to css

File: src/components/Footer/Footer.css (hunk 1 above)
Comment (yazfield) [reply src/components/Footer/Footer.css:1]: Don't put compiled files in the Git repo.
The repo should only contain .scss files, they will be automatically compiled to css

**Pull request merged**